drivers/storage/portworx/component/pvccontroller.go

```diff
@@ -333,13 +342,17 @@ func (c *pvcController) createDeployment(
 			command = append(command, "--port="+port)
 		} else if pxutil.IsAKS(cluster) {
 			command = append(command, "--port="+AksPVCControllerInsecurePort)
+		} else if c.isK3sDeployment() {
```

Comment:
Once we use the same constant name, we can club these two conditions into one condition with an OR operator.